src/Widgets/AppEntry.vala

```diff
@@ -42,26 +52,55 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
             name = _("Other");
         }
 
+        var image = new Gtk.Image.from_icon_name ("pan-end-symbolic", SMALL_TOOLBAR);
+        image.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
         var label = new Gtk.Label (name) {
             hexpand = true,
             xalign = 0
         };
         label.get_style_context ().add_class (Granite.STYLE_CLASS_H3_LABEL);
 
+        var expander_content = new Gtk.Box (HORIZONTAL, 3);
+        expander_content.add (image);
+        expander_content.add (label);
+
+        expander = new Gtk.ToggleButton () {
+            child = expander_content,
+            can_focus = false
```

Comment:
I believe the more recent pattern is to let buttons have focus, for visual confirmation when navigating with the keyboard. but i would like a confirmation from someone from the UX team.

Reply:
@elementary/ux
Btw it seems I can't request a review from ux in this repo?

Reply:
Yeah it's generally not a good idea not to allow a widget to have focus since that makes it completely inaccessible via keyboard